homework/index.js

```diff
@@ -30,14 +30,93 @@
     return elem;
   }
 
+  function createOption(repositories, select) {
+    createAndAppend('option', select, { text: 'Select a repository', disabled: 'disabled' });
+    repositories
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .forEach(repository => {
+        createAndAppend('option', select, { text: repository.name, value: repository.name });
+      });
+  }
+
+  function renderRepoBasisInfo(repository, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repository}`;
+    const container = document.getElementById('root');
+    fetchJSON(url, (err, repo) => {
+      if (err) {
+        createAndAppend('div', container, { text: err.message, class: 'alert-error' });
+      }
+      ul.innerHTML = '';
```

Comment:
This `ul` with a single `li` that contains just `table` doesn't make much sense here. Why do we need a list? This the HTML you generate:

```html
<div class="info-box">
  <ul class="info-ul">
    <li class="info-list">
      <table>
        <tbody>
          <tr>
            <th>Repository:</th>
            <td><a href="https://github.com/HackYourFuture/alumni" target="_blank">alumni</a></td>
          </tr>
          <tr>
            <th>Description:</th>
            <td>Alumni Guides, everything you need to land a job</td>
          </tr>
          <tr>
            <th>Forks:</th>
            <td>9</td>
          </tr>
          <tr>
            <th>Updated at:</th>
            <td>2019-06-21T11:38:21Z</td>
          </tr>
        </tbody>
      </table>
    </li>
  </ul>
</div>
```